solutions/usaco-380.mdx

```diff
@@ -131,5 +131,118 @@ public class CrossCountrySkiing {
 
 </JavaSection>
 
+<CPPSection>
+
+```cpp
+// @author: ascyrax
+
+#include <bits/stdc++.h>
+
+using namespace std;
+
+#define endl "\n"
+
+#define pb push_back
+
+int r, c;
+vector<vector<int>> grid;     //grid stores the elevation
+vector<vector<bool>> visited; //this will be true if the corresponding location of the grid has been visited atleast once
+
+//right bottom left top
+int drow[4] = {0, 1, 0, -1};
```

Comment:
to standardize this variable name it would be nice to call them dx, dy but you don't have to